test/e2e/performanceprofile/functests/11_mixedcpus/mixedcpus.go

```diff
@@ -175,6 +175,30 @@ var _ = Describe("Mixedcpus", Ordered, func() {
 			Expect(err).ToNot(HaveOccurred())
 			Expect(cfg.Quota).To(Or(Equal("max"), Equal("-1")))
 		})
+		It("should have OPENSHIFT_ISOLATED_CPUS and OPENSHIFT_SHARED_CPUS env variables under the container", func() {
+			rl := &corev1.ResourceList{
+				corev1.ResourceCPU:    resource.MustParse("1"),
+				corev1.ResourceMemory: resource.MustParse("100Mi"),
+				sharedCpusResource:    resource.MustParse("1"),
+			}
+			p, err := createPod(ctx, testclient.Client, testutils.NamespaceTesting,
+				withRequests(rl),
+				withLimits(rl),
+				withRuntime(components.GetComponentName(profile.Name, components.ComponentNamePrefix)))
+			Expect(err).ToNot(HaveOccurred())
+			cmd := printMixedCPUsEnvCmd()
+			output, err := pods.ExecCommandOnPod(testclient.K8sClient, p, "", cmd)
+			Expect(err).ToNot(HaveOccurred(), "failed to execute command on pod; cmd=%q pod=%q", cmd, client.ObjectKeyFromObject(p).String())
+			isolatedAndShared := strings.Split(string(output), "\r\n")
+			isolated, err := cpuset.Parse(isolatedAndShared[0])
```

Comment:
addressing unknown elements can hit nil pointer, I believe we need to verify (expect()) that the length of isolatedAndShared is 2 and fail if not.

Reply:
We should use `mustParse` here, i'll fix that